.github/workflows/release.yml

```diff
@@ -0,0 +1,85 @@
+name: Release Wave
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release Version'
+        required: true
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  VERSION: "${{ github.event.inputs.version }}"
+  TAG: "v${{ github.event.inputs.version }}"
+
+jobs:
+  ci-test:
+    name: CI Test
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build
+        run: make all
```

Comment:
Is this really necessary since we run just ui unit tests here? What do you think about just doing `cd ui && npm run ci` ?

Reply:
@mturoci Yes that's enough. I will change it. Thanks!